docs/administration-guide/operation-score.md

```diff
@@ -0,0 +1,87 @@
+# Operation Scores
+
+!!! info "Under construction"
+    This page is currently under construction and will be expanded on in future.
+
+To use the `OperationChainLimiter` GraphHook then you will need to configure that GraphHook to use the `NamedOperationScoreResolver`. 
+This will then allow you to have custom scores for each NamedOperation.
+
```

Comment:
Is it maybe worth mentioning that OperationScores work like a credit and cost system where each Operation has a  score (cost) and each user of the system can have an amount of score (credits) they can spend to run Operations in one go.

Reply:
I agree, but I think for this PR, this file is only intended to have minimal content which was taken from the named operations page. This recommendation can be incorporated in the operation score PR. It may be better to not make any changes to this file in this PR and do all of it in the operation score PR?